src/pages/Account/Components/CoinsTable.tsx

```diff
@@ -64,62 +61,79 @@ function CoinTypeCell({
 
   return (
     <GeneralTableCell sx={{width: 450}}>
-      <HashButton hash={assetType} type={getType()} size="large" />
+      <HashButton
+        hash={data.tokenAddress ?? data.faAddress ?? "Unknown"}
+        type={getType()}
+        size="large"
+        img={data.logoUrl}
+      />
     </GeneralTableCell>
   );
 }
 
-export function CoinsTable({
-  coins,
-}: {
-  coins: {
-    name: string;
-    amount: number;
-    decimals: number;
-    symbol: string;
-    assetType: string;
-    assetVersion: string;
-  }[];
-}) {
+function CoinVerifiedCell({data}: {data: CoinDescriptionPlusAmount}) {
+  return VerifiedCoinCell({
+    data: {
+      id: data.tokenAddress ?? data.faAddress ?? "Unknown",
+      known: data.isInPanoraTokenList,
+      isBanned: data.isBanned,
+      isInPanoraTokenList: data.isInPanoraTokenList,
+    },
+  });
+}
+
+export type CoinDescriptionPlusAmount = {
+  amount: number;
+  tokenStandard: string;
+} & CoinDescription;
+
+export function CoinsTable({coins}: {coins: CoinDescriptionPlusAmount[]}) {
+  // TODO: For FA, possibly add store as more info
   return (
     <Table>
       <TableHead>
         <TableRow>
           <GeneralTableHeaderCell header="Name" />
+          <GeneralTableHeaderCell header="Asset Type" />
+          <GeneralTableHeaderCell header="Asset" />
+          <GeneralTableHeaderCell
+            header="Verified"
+            tooltip={
+              <LearnMoreTooltip
+                text="This uses the Panora token list to verify authenticity of known assets on-chain.  It does not guarantee anything else about the asset and is not financial advice."
+                link="https://github.com/PanoraExchange/Aptos-Tokens"
+              />
```

Comment:
If you wouldn't mind modifying `LearnMoreTooltip` to add `target="_blank" to the `Link`. That way we open these external links in new tabs.